stargate/pkg/auth/interface.go

```diff
@@ -0,0 +1,5 @@
+package auth
+
+type AuthProviderIFace interface {
```

Comment:
nit: consider dropping `IFace`.

Reply:
I'm not a big fan of the suffix either but I see it quite a bit in other golang projects. I can go ahead and drop it.

Reply:
Removed